### ISSUE TITLE
Add static validation test for events.js

### ISSUE THREAD

**Issue by rrudol** [akai-org/hackathons#13]
## Description
A small typo can destroy build. Please add a simple test that will validate the correctness of the event.js file.

**Comment by roertbb** [comment]
How would you like to achieve that? I'd suggest some Github Action running test on pull request. Do you have any other idea?

**Comment by rrudol** [comment]
Exactly, let's prepare regular Jest test case which just loads and parse data and then let's run it using Github Actions 👍
